assignments/assignment_1/index.js

```diff
@@ -7,12 +7,11 @@ function getNameFromCommandLine() {
 
 function getNameFromEnv() {
     // Write your code here
-    process.env.name = "Pardhu";
+    process.env.name = "Yash";
```

Comment:
no need to declare the environment var "name" here it is already declared in the environment for test cases, just returning is sufficient.